connectors/v2/dreamfm.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+/**
+ * remove zero width characters & trim
+ * @param  {string} text to clean up
+ * @return {string} cleaned up text
+ */
+function cleanText(input) {
+	if (input === null) {
+		return input;
+	}
+	return input.replace(/[\u200B-\u200D\uFEFF]/g, '').trim();
+}
+
+function getArtist() {
+	var artist = $('#tracka').text() ||
+				null;
+	return cleanText(artist);
+}
+
+function getTrack() {
+	var track = $('#tracktitle').text() ||
+				null;
+	return cleanText(track);
+}
+
+/* @returns {{artist, track}} */
+Connector.getArtistTrack = function () {
```

Comment:
This method is not really needed since it is called only when either getArtist or getTrack returns an empty result.

/blob/master/core/content/connector.js#L154
/blob/master/core/content/connector.js#L263